### DISCUSSION TITLE
Docker compose example in readme

### DISCUSSION THREAD

**Discussion by Motophan** [guydavis/machinaris#441] [Q&A] [answered]
Docker compose example in readme would help me get started. 

I am very interested in a IO exaustion scenario to have chia be prioritized over the other forks. Is there stanza i can add in docker compose to do this? 

Thanks.

**Comment by guydavis** [comment, marked as answer]
Good day! Please try the `docker-compose.yml` as generated using the [app launch wizard](https://www.machinaris.app/).  Please start with the default, un-edited, `docker-compose.yml` to ensure you launch Machinaris correctly to begin with. 

Once successfully installed and configured, you can look at changing the default `docker-compose.yml` , for example by adding [resource constraints](https://docs.docker.com/compose/compose-file/compose-file-v3/#resources) per container.  Depending on how you choose to [blockchain sync](https://github.com/guydavis/machinaris/wiki/Blockchains) (Chia + forks), expect initially high usage.  Things settle down when farming a synced blockchain though.

Please mark this question as Answered if this addressed your ticket.  Thanks!

**Reply by guydavis** [reply]
Marking this as Answered as no reply.
